lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -453,8 +455,16 @@ class AssertOnlyCertificate(Certificate):
     def __init__(self, module):
         super(AssertOnlyCertificate, self).__init__(module)
         self.signature_algorithms = module.params['signature_algorithms']
-        self.subject = module.params['subject']
-        self.issuer = module.params['issuer']
+        if module.params['subject']:
+            self.subject = crypto_utils.parse_name_field(module.params['subject'])
+        else:
+            self.subject = []
+        self.subject_strict = False
```

Comment:
I understand the need for the _strict param, but why not using the same pattern as for `self.keyUsage_strict = module.params['keyUsage_strict']` here? Is it expected to be done in a subsequent review ?

Reply:
I want to do that in the next upcoming PR, I didn't want to write everything at once and then have 3 different things in one PR.

Reply:
Ack.

Reply:
see #34090